Simulation/SimulationMessaging.cs

```diff
@@ -77,6 +79,28 @@ public static class SimulationMessaging
         static private List<ConditionalRSL> _attributeConditionalRSL;
         [ThreadStatic]
         static private Jurisdiction _jurisdiction;
+        [ThreadStatic]
+        static private List<RemainingLife> _attributeRemainingLife;
+        [ThreadStatic]
+        static private Dictionary<string,string> _noTreatmentRemainingLife;
+        [ThreadStatic]
+        static private List<BudgetCriteria> _budgetCriteria;
+
+        public static void ReleaseSimulationData()
```

Comment:
This will likely need to be made [ThreadStatic]

Reply:
All static members of the simulation messaging class need to be checked to see if they need to be ThreadStatic.